Source/DafnyLanguageServer/Language/Symbols/LspSymbolGeneratingVisitor.cs

```diff
@@ -70,21 +70,19 @@ public class LspSymbolGeneratingVisitor : ISymbolVisitor<IEnumerable<DocumentSym
 
     private IEnumerable<DocumentSymbol> CreateSymbolsOfEntryDocument(ILocalizableSymbol symbol, Boogie.IToken token, SymbolKind kind) {
       var children = symbol.Children.SelectMany(Visit);
-      if (!IsPartOfEntryDocument(token)) {
+      if (!IsPartOfEntryDocument(token) || token.line == 0) {
```

Comment:
I'm confused. `IsPartOfEntryDocument` is defined as 

`token.line != 0 && _symbolTable.CompilationUnit.Program.IsPartOfEntryDocument(token)`

so `!IsPartOfEntryDocument(token)` should be 

`token.line == 0 || !_symbolTable.CompilationUnit.Program.IsPartOfEntryDocument(token))`

Wouldn't that make `|| token.line == 0` obsolete?

Reply:
Good catch, you're absolutely right. I've removed the duplicated check.